slack-lint/src/main/java/slack/lint/resources/model/RootIssueData.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2022 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package slack.lint.resources.model
+
+import com.android.tools.lint.detector.api.Location
+
+data class RootIssueData(val alias: String, val nameLocation: Location)
+
+val IMPORT_ALIASES =
+  mapOf(
+    "slack.l10n.R" to "L10nR",
+    "slack.uikit.resources.R" to "SlackKitR",
+    "slack.uikit.R" to "UiKitR",
+    "slack.coreui.R" to "CoreUiR"
+  )
```

Comment:
This is fairly slack-specific, let's make these a configured property instead.

Reply:
Any examples of how to do this?

Reply:
https://github.com/slackhq/slack-lints/blob/main/slack-lint/src/main/java/slack/lint/inclusive/InclusiveNamingChecker.kt#L72

Reply:
451f2d67c132a2e70dbfb8ee80b527b2724bc3a5